react-components/src/apps/admin/AllStats/AppStatsComponents/NavBarTabs.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * Imported in AllStats.js. This function imports the files for all the tabs that are present under TabPanels
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import AppBar from "@material-ui/core/AppBar";
+import Tabs from "@material-ui/core/Tabs";
+import Tab from "@material-ui/core/Tab";
+import Typography from "@material-ui/core/Typography";
+import Box from "@material-ui/core/Box";
+import JobsTab from "./JobsTab";
+import AppsTab from "./AppsTab";
+import UsersTab from "./UsersTab";
+import ids from "./AllStatsIDs";
+import { getMessage, withI18N } from "@cyverse-de/ui-lib";
+import myMessagesFile from "./messages.js";
+import build from "@cyverse-de/ui-lib/src/util/DebugIDUtil";
+
+function TabPanel(props) {
```

Comment:
Although there are some nice things about the example code. I do like `a11yProps` (though we may want to `build` a debug ID in there).

I also wonder if this version might be more performant than our simple version, where we change the contents that are rendered based on the `tabIndex` 🆚 this version that renders all the contents and hides all of them except the currently selected index. It's possible that our old version could thrash the DOM more. 🤔

That said, I think we don't need the `LinkTab` components that imitate links (we don't have routing that allows the user to navigate our app with the browser's back/forward buttons). So maybe stick with the [Simple Tabs](https://material-ui.com/components/tabs/#simple-tabs) example.

Reply:
🤔 good point, it might actually be more performant which would explain the change in their docs.  Agree with sticking with `Simple Tabs` and getting rid of `LinkTab`.

Reply:
@psarando What is a debug ID?

Reply:
Oh, sorry, that's what they are called in GWT. I just meant a static ID, but dynamically built with the ui-lib's DebugIDUtil `build` function.

Reply:
So `a11yProps` could return something like `id: build(baseId, ids.NAV_TAB, index)`.

Reply:
I've made the necessary changes.